frontend/src/metabase/dashboard/components/Dashboard/stickyParameters.unit.spec.js

```diff
@@ -79,4 +87,50 @@ describe("updateParametersWidgetStickiness", () => {
 
     expect(setState).not.toHaveBeenCalled();
   });
+
+  describe("on small device", () => {
+    beforeEach(() => {
+      simulateSmallScreen();
+    });
+
+    it("makes filters not sticky if too many parameters", () => {
```

Comment:
I think you should specify in the test name that if there are more than 6 parameters (7 or more), the parameters header won't be sticky. Since this is the previous behavior we have before this PR.

https://github.com/metabase/metabase/blob/89dc0f3f4d87751691260e59e76d34e92d3e939b/frontend/src/metabase/dashboard/components/Dashboard/stickyParameters.js#L3

Reply:
but if we update this number, the test name will become obsolete

Reply:
I used `MAXIMUM_PARAMETERS_FOR_STICKINESS` in the name, should be good

Reply:
Fair enough. But I think `MAXIMUM_PARAMETERS_FOR_STICKINESS` is already the behavior. I mean if you change the `MAXIMUM_PARAMETERS_FOR_STICKINESS` I'd expect the test to even fail. `MAXIMUM_PARAMETERS_FOR_STICKINESS` is kind of a requirement we're testing.

I normally create a new variable that matches the constant used in our code in the test to make sure we test this behavior.

Let's say this is some number from a project that products already specify, if we accidentally change the number, I'd expect some form of test to catch that error, but if we use that same constant in the test, we wouldn't have caught it.

Maybe we could discuss this in the next FE sync.